rct229/rules/section5/section5rule18.py

```diff
@@ -101,24 +105,35 @@ def get_calc_vals(self, context, data=None):
                 # all building segments in AreaType rule has the same area type
                 # (see create_context_list function in the parent class)
                 area_type = building_segments_b[0]["area_type_vertical_fenestration"]
-                area_type_wwr = 0.0
                 area_type_target_wwr = 0.0
-                if area_type is not "NONE":
-                    area_type_wwr = (
-                        area_type_window_wall_ratio_b[area_type]["total_window_area"]
-                        / area_type_window_wall_ratio_b[area_type]["total_wall_area"]
-                    )
+                area_type_wwr = (
+                    area_type_window_wall_ratio_b[area_type]["total_window_area"]
+                    / area_type_window_wall_ratio_b[area_type]["total_wall_area"]
+                )
+                if area_type != "NONE":
```

Comment:
Is there an area_type called "NONE"?  There is an "OTHER", but we have already filtered those out.

Reply:
The NONE is not an `area_type` from the schema, but is a [key ](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/get_area_type_window_wall_areas.md?plain=1#L27)returned from the `area_type_window_wall_areas` when dealing with data missing.

In this case, we just have to flag it out for manual checking.